test/data/services/test_isolines.py

```diff
@@ -0,0 +1,376 @@
+# -*- coding: utf-8 -*-
+
+"""Unit tests for cartoframes.data.services.Isolines"""
+import unittest
+import os
+import sys
+import json
+import warnings
+import pandas as pd
+
+from carto.exceptions import CartoException
+
+from cartoframes.data import Dataset
+from cartoframes.auth import Credentials
+from cartoframes.utils.columns import normalize_name
+
+
+from cartoframes.data.clients import SQLClient
+
+
+from cartoframes.data.services import Isolines
+
+
+try:
```

Comment:
You can remove the dependency and try/catch because it is unused

Reply:
Well, it is actually used to mark the names of temporary tables created for tests with `gpd` if geopandas is available. This is something I copied from the Dataset tests without really knowing what it can be useful for. Is it to run concurrent tests with/without geopandas? to analyse failing tests? ¯\_(ツ)_/¯

Invoking the apparent author of the original code to see if he can shed some light: @alrocar should we keep that geopandas nonsense?

Reply:
We are using the `geopandas` dep in different ways, but in the `test_dataset`
```
gdf = geopandas.GeoDataFrame(df)
```
About the try-catch, the idea is to use geopandas only if it is installed (because it is not right now)  and basically we are adding the import in this way to avoid problems if it is not installed.